almdrlib/client.py

```diff
@@ -244,7 +244,20 @@ def __init__(self, required=False, description=None, session=None):
         self._session = session
         self._content_types = {}
         self._content = {}
-        self._default_content_type = None
+        self._default_content_type = False
```

Comment:
This was unused previously.  `None` will represent "there is no default" and `False` will represent "the default isn't set yet."